### PR TITLE
Relation sorting

### PR DESCRIPTION
**Pull request by Nathan-O**
New pull request pointing to the correct version branch. 

@Samsinite can you give this another review?

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,5 +10,6 @@ group :test do
   gem 'combustion', '~> 0.5.2'
   gem 'rack-test'
   gem 'pundit'
-  gem 'active_model_serializers', '~> 0.8.0'
+  # gem 'active_model_serializers', '~> 0.8.0'
+  gem 'active_model_serializers', '~> 0.10.0'
```
Comment (Samsinite) [inline Gemfile]: 0.7.0 is on 0.8.0, so this needs bumbped back down for this PR

File: api_me.gemspec [hunk 2]
```diff
@@ -21,7 +21,8 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'activerecord',             '>= 3.2.0'
   s.add_runtime_dependency 'activesupport',            '>= 3.2.0'
   s.add_runtime_dependency 'pundit',                   '~> 1.0'
-  s.add_runtime_dependency 'active_model_serializers', '~> 0.8.0'
+  # s.add_runtime_dependency 'active_model_serializers', '~> 0.8.0'
+  s.add_runtime_dependency 'active_model_serializers', '~> 0.10.0'
```
Comment (Samsinite) [inline api_me.gemspec]: Ditto

**Comment by Nathan-O** [comment]
Righto @Samsinite got that reverted to 0.8.0, and I apoligixe, I could have sworn I did that after we talked about it the other day. Must have spaced it.

**Comment by Samsinite** [comment]
Looks like the tests are failing due to `bundle install` failing. Probably just needs ran again

**Comment by Samsinite** [comment]
Restarted the build

**Comment by Samsinite** [comment]
Looks like the tests are failing

**Comment by Samsinite** [comment]
Ping me when this is merged and I'll run a rake release on this branch.

**Pull request merged**